dojo/templates/dojo/paging_snippet.html

```diff
@@ -28,21 +28,21 @@
                     <div class="btn-group">
-                        <button type="button" class="btn-sm btn-secondary dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                        <button data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" aria-label="{% trans 'Page size selector' %}" id="pageSizeDropdown" class="btn-sm btn-secondary dropdown-toggle" type="button">
                             {% trans "Page Size" %}
                             <span class="caret"></span>
                             <span class="sr-only">{% trans "Toggle Dropdown" %}</span>
                         </button>
-                        <ul class="dropdown-menu">
-                            <li><a href="?{% url_replace request page_size_param 25 %}">25</a></li>
-                            <li><a href="?{% url_replace request page_size_param 50 %}">50</a></li>
-                            <li><a href="?{% url_replace request page_size_param 75 %}">75</a></li>
-                            <li><a href="?{% url_replace request page_size_param 100 %}">100</a></li>
-                            <li><a href="?{% url_replace request page_size_param 150 %}">150</a></li>
+                        <ul class="dropdown-menu" role="menu" aria-labelledby="pageSizeDropdown">
+                            <li role="presentation"><a href="?{% url_replace request page_size_param 25 %}" aria-label="{% trans '25 items per page' %}" role="menuitem">25</a></li>
+                            <li role="presentation"><a href="?{% url_replace request page_size_param 50 %}" aria-label="{% trans '50 items per page' %}" role="menuitem">50</a></li>
+                            <li role="presentation"><a href="?{% url_replace request page_size_param 75 %}" aria-label="{% trans '75 items per page' %}" role="menuitem" >75</a></li>
+                            <li role="presentation"><a href="?{% url_replace request page_size_param 100 %}" aria-label="{% trans '100 items per page' %}" role="menuitem">100</a></li>
+                            <li role="presentation"><a href="?{% url_replace request page_size_param 150 %}" aria-label="{% trans '150 items per page' %}" role="menuitem">150</a></li>
                             {% if page.paginator.count > 500 %}
-                                <li><a href="?{% url_replace request page_size_param 500 %}">500</a></li>
+                                <li role="presentation"><a href="?{% url_replace request page_size_param 500 %}" aria-label="{% trans '500 items per page' %}" role="menuitem" >500</a></li>
                             {% endif %}
-                            <li><a href="?{% url_replace request page_size_param page.paginator.count %}">{% trans "All" %}</a></li>
+                            <li role="presentation"><a href="?{% url_replace request page_size_param page.paginator.count %}" aria-label="{% trans 'All items per page' %}" role="menuitem">{% trans "All" %}</a></li>
```

Comment:
Probably more like this
```suggestion
                            <li role="presentation"><a href="?{% url_replace request page_size_param page.paginator.count %}" aria-label="{% trans 'All items on one page' %}" role="menuitem">{% trans "All" %}</a></li>
```